src/libs/actions/BankAccounts.js

```diff
@@ -822,7 +832,7 @@ function setupWithdrawalAccount(data) {
 }
 
 function hideBankAccountErrors() {
-    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {error: '', existingOwnersList: ''});
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {error: '', existingOwnersList: '', errors: {}});
```

Comment:
Probably you'll get conflicts if my other PR is merged, but I noticed that `existingOwnersList` [isn't actually used anywhere](https://github.com/Expensify/App/search?q=existingOwnersList) but `existingOwners` is - so let's make sure this bug doesn't get reintroduced here.

Reply:
Should I change `existingOwnersList` for `existingOwners` now? or we handle it once there is conflict?

I didn't know you could do a search like that in Github, cool feature.

Reply:
I guess it doesn't matter yet, but just pointing out the code you have here does nothing so probably we should update it.

Reply:
Are we actually clearing the errors if we do `errors: {}`? This will need to be `errors: null` I think?

Reply:
> Are we actually clearing the errors if we do `errors: {}`? This will need to be `errors: null` I think?

You are right, I was clearing this the wrong way. 🤦

Reply:
Actually, I'm just going to set it to `null` and the will clear the key. When we get the errors we get them using `lodashGet(this.props, ['reimbursementAccount', 'errors'], {})`. I feel like setting this to `null` and then `{}` is uglier and doesn't make things much better.